private_routing.tf

```diff
@@ -0,0 +1,50 @@
+#-------------------------------------------------------------------------------
+# Note that all these resources depend on the VPC, the NAT GWs, or
+# both, and hence on the
+# aws_iam_role_policy_attachment.provisionnetworking_policy_attachment
+# resource.
+# -------------------------------------------------------------------------------
+
+#-------------------------------------------------------------------------------
+# Set up routing for the private subnets.
+#
+# The public subnets will use the default routing table in the VPC, as
+# defined in public_routing.tf.
+# -------------------------------------------------------------------------------
+
+# Each private subnet gets its own routing table, since each subnet
+# uses its own NAT gateway.
+resource "aws_route_table" "private_route_tables" {
+  for_each = toset(var.private_subnet_cidr_blocks)
+
+  tags   = var.tags
+  vpc_id = aws_vpc.the_vpc.id
+}
+
+# Route all non-local COOL (outside this VPC but inside the COOL)
+# traffic this VPC through the transit gateway.
```

Comment:
I think you meant to delete "this VPC" from this comment?

Reply:
Fixed in commit 9c5e061.